docs/deploy/howto-deploy-all-in-one.md

```diff
@@ -14,10 +14,10 @@ This article explains how to deploy the Azure Industrial IoT Platform and Simula
 
 The platform and simulation can also be deployed using the deploy script.
 
-1. If you have not done so yet, clone the GitHub repository. To clone the repository, you need git. If you do not have git installed on your system, follow the instructions for [Linux or Mac](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), or [Windows](https://gitforwindows.org/) to install it. Open a new command prompt or terminal and run:
+1. If you have not done so yet, clone the GitHub repository from one of our release branches. To clone the repository, you need git. If you do not have git installed on your system, follow the instructions for [Linux or Mac](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), or [Windows](https://gitforwindows.org/) to install it. Open a new command prompt or terminal and run:
 
    ```bash
-   git clone https://github.com/Azure/Industrial-IoT
+   git clone -b <ReleaseBranchName> https://github.com/Azure/Industrial-IoT
```

Comment:
Shall we use `--branch` instead of `-b` to be more explicit ?

Reply:
I think this is covered.